components/builder-sessionsrv/README.md

```diff
@@ -1,3 +1,10 @@
 # builder-sessionsrv
 
 Generates, validates, and invalidates authenticated sessions for builder clients
+
+# Adding new migrations
+
+```
+cargo install diesel-cli
+diesel migration generate --migration-dir ./src/migrations <migration_name>
```

Comment:
Where should the new sql be added?  Would be good to have this section be a bit more detailed on steps and practices devs should follow..

Reply:
Should we have the same documentation for each service? I forgot I documented this in the sessionsrv readme but not for originsrv or jobsrv. They would likely all be the same.